implementation/eCatalogue/shacl_shapes/eCatalogue_shapes.ttl

```diff
@@ -238,7 +231,6 @@ cat-shape:epo-cat-Catalogue-epo-cat-comprisesCatalogueLine a sh:PropertyShape ;
     sh:sparql [ sh:select "SELECT ?this ?that WHERE { ?this <http://data.europa.eu/a4g/ontology#comprisesCatalogueLine> ?that . ?that <http://data.europa.eu/a4g/ontology#comprisesCatalogueLine> ?this .}" ] .
 
 cat-shape:epo-cat-Catalogue-epo-cat-hasActivityCode a sh:PropertyShape ;
```

Comment:
This PropertyShape (cat-shape:epo-cat-Catalogue-epo-cat-hasActivityCode) got 'relaxed', it says the same as cat-shape:epo-cat-CatalogueLine-epo-cat-hasActivityCode below. 

If this is no longer part of the ontology both should go away